File: 2MONTH_LESSON1-master/app/src/main/java/com/example/a2monthlesson1/SecondActivity.java
package com.example.a2monthlesson1;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.TextView;

public class SecondActivity extends AppCompatActivity {

    private TextView textView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_second);

        textView = findViewById(R.id.tv_result);

        String result = getIntent().getStringExtra(MainActivity.USERNAME_KEY);

        textView.setText(result);
    }
}

File: 2MONTH_HW5-master/app/src/main/java/com/example/a2_monthlesson5/SecondActivity.kt
package com.example.a2_monthlesson5

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.TextView

class SecondActivity : AppCompatActivity() {

    lateinit var emailTextView : TextView
    lateinit var numberTextView : TextView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_second)

        emailTextView = findViewById(R.id.tv_email)
        numberTextView = findViewById(R.id.tv_number)

        val email = intent.getStringExtra("email")
        emailTextView.text = email
        val number = intent.getStringExtra("number")
        numberTextView.text = number
    }
}